src/planscape/base/condition_types.py

```diff
@@ -50,6 +50,8 @@ class Metric(TypedDict):
     display_name: NotRequired[str]
     current_conditions_only: NotRequired[boolean]
     colormap: NotRequired[str]
+    min_value: NotRequired[int]
```

Comment:
should these classes/structs also be updated with the data source related fields? and should min/max value be added in Element/Pillar as well?

Reply:
Min/max aren't needed for elements and pillars, since they are always -1/1.

I don't think the data source fields are needed in the backend, just the frontend.  Can I add those in a separate PR if they're needed?

Reply:
sg, let's wait and see